README.md

```diff
@@ -0,0 +1,264 @@
+</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo
+
+[RENTCAR-Luxury Cars](https://capstone-rails.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📋 Kanban Board 
+
+We have used a Kanban board to split the tasks and simplify the project workflow.
+
+### [Kanban Board](https://github.com/roniy68/capstone-rails/projects/1).
+
+### The members: [@Salwa](https://github.com/Salwa99), [@Ahmed](https://github.com/roniy68), [@Talha](https://github.com/Talha-Hanif5666)
+
+
+<img src="https://github.com/roniy68/capstone-rails/assets/77187003/bdb6c83b-1e4c-4d1b-98ed-cf28d59ed7f2" width=600 height=400 alt="kanban board" />
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Tech Stack  <a name="tech-stack"></a>
+
+- **Back-end** 💻:   
+  <details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://redux-toolkit.js.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+  - **Front-end** 💻: 
+    <details>
+    <summary>Client</summary>
+    <ul>
+      <li><a href="https://react.dev/">React</a></li>
+      <li><a href="https://redux-toolkit.js.org/">Redux Toolkit</a></li>
+      <li><a href="https://reactrouter.com/en/main">React Router</a></li>
+      <li><a href="https://tailwindcss.com/">Tailwind</a></li>
+      <li><a href="https://reactrouter.com/en/main">React icons</a></li>
+    </ul>
+</details>
+
+
+- **Tools** 🛠: 
+  - [Rspec](https://rspec.info/)
+  - [Esbuild](https://esbuild.github.io/)
+  - [Render](https://render.com/)
+
+
+<details>
+<summary>Linter</summary>
+  <ul>
+    <li>Eslint</li>
+    <li>Rubocop</li>
+    <li>Stylint</li>
+  </ul>
+</details>
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ✨ Key Features  <a name="key-features"></a>
+
+- **Log in with a username to access the palteform.**
+- **Browse list of available cars.**
+- **Add your cars or delete them.**
+- **Reserve a cars.**
+- **Responsive design.**
+- **Use API endpoint.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📘 Getting Started  <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### 📋 Prerequisites
+
+Make sure you have the following installed on your machine:
+- [Ruby 3.1.3 or higher](https://www.ruby-lang.org/en/)
+- [Rails 7.0.4 or higher](https://rubyonrails.org/)
+- [PostgreSQL 15.2 or higher](https://www.postgresql.org/)
+- [Node 18.14 or higher](https://nodejs.org/en)
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired foler.
+
+```sh
+  git clone https://github.com/roniy68/capstone-rails.git
+```
+
+```sh
+  cd capstone-rails
+```
+
+### 📥 Installation
+Install the required gems with:
+
+```sh
+bundle install
+```
+
+Install the node dependencies with:
+
+```sh
+yarn install
+```
+
+```sh
+npm install
+```

```

Comment:
- [ ] OPTIONAL:  Be careful here.  Are both npm and yarn needed?  Or just one of them?  

![image](https://github.com/roniy68/capstone-rails/assets/57421823/4b5b48b4-fd7a-4091-b04e-a8281699fadd)

Since your project works and may depend on both, I leave this as and optional heads-up. 👍

Reply:
🔥 Cleared The Confusion with or